src/index.html

```diff
@@ -7,8 +7,73 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+    rel="preconnect"
+    href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="container">
+        <a class="img" href="#home">
+          <img
+            class="logo"
+            src="images/logo.png"
+            alt="logo">
+        </a>
+        <nav>
+          <ul class="navList">
+            <li class="navItem">
+              <a class="link is-active" href="#apple">
+                Apple
+              </a>
+            </li>
+            <li class="navItem">
+              <a class="link" href="#samsung">
+                Samsung
+              </a>
+            </li>
+            <li class="navItem">
+              <a class="link" href="smartphones">
```

Comment:
```suggestion
              <a class="link" href="#smartphones">
```
<img width="1337" alt="image" src="https://user-images.githubusercontent.com/102413088/192535948-6c3fa25b-c692-40e5-8be8-5f5e566d12fc.png">